index.html

```diff
@@ -1,29 +1,195 @@
 <!DOCTYPE html>
-<html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.cdnfonts.com/css/anurati" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.typekit.net/xse6ume.css">
+    <title>Assignment 2</title>
+</head>
+<body>
+    <!-- Header -->
+    <div class="header">
+        <h1>OUR ECOLOGY</h1>
+        <p><a href="#">Home</a></p>
+        <p><a href="#mission">Mission</a></p>
+        <p><a href="#">Community</a></p>
+        <p><a href="#news">News</a></p>
+        <p><a href="#">About Us</a></p>
+        <p><a href="#contact">Contact</a></p>
+        <p><a href="#donate">Donate</a></p>
+        <!-- Search bar text and icon -->
+            <a><input type="text" placeholder="Search"></a>
+                <img src="search.png" alt="Search icon" width="30">
+    </div>
```

Comment:
This content should be in a `header` element to add semantic markup. 

Have a look at some of our exercises at the start of the course for examples on this. 

```html
<header>
   ...
</header>
```